### PR TITLE
minimum platform up to iOS 13

### PR DESCRIPTION
**Pull request by charlessnow**

### CODE REVIEW

**Comment by charlessnow** [comment]
iOS v13

File: Package.swift [hunk 1]
```diff
@@ -112,7 +112,7 @@ let permissionsTargets: [Target] = [
 let package = Package(
     name: "PermissionsSwiftUI",
     defaultLocalization: "en",
-    platforms: [.iOS(.v11)],
+    platforms: [.iOS(.v13)],
```
Comment (charlessnow) [inline Package.swift:115]: minimum platform up to iOS 13

**Review by charlessnow — COMMENTED** [review]
minimum platform up to iOS 13

**Review by jevonmao — APPROVED** [review]
Thank you :)

**Pull request merged**